tensorboard/webapp/runs/views/runs_table/runs_table_component.scss

```diff
@@ -16,33 +16,28 @@ limitations under the License.
 
 // 42px + 1px for border.
 $_table-content-row-min-height: 43px;
+$_font-size: 13px;
 
 :host {
+  background-color: #fff;
   display: block;
+  font-size: $_font-size;
   overflow: hidden;
-  background-color: #fff;
 }
 
 .table-container {
   contain: layout paint;
+  height: 100%;
```

Comment:
[No action needed necessarily]

Let's make sure that this works internally.  For RunsTableContainers that use `useFlexibleLayout=false` in an experiment list type of view, a runs table's height should grow depending on the number of rows within.

Reply:
I did check both internally/externally.